README.md

```diff
@@ -142,7 +142,7 @@ This example outputs:
 
 ### Auto detection of the instruction set extension to be used
 
-The same computation operating on vectors and using the most performant instruction set available:
+The same computation operating on vectors and using the most performant instruction set available at compile time, absed on the provided compiler flags (e.g. ``-mavx2`` for GCC and Clang to target AVX2):
```

Comment:
we could add a reference to [this](https://gitlab.com/x86-psABIs/x86-64-ABI) and maybe suggest the arch levels to the user?

Reply:
I provide an example here: https://github.com/DiamonDinoia/random